codegen/src/test/resources/software/amazon/awssdk/codegen/poet/rules/endpoint-parameters.java

```diff
@@ -44,53 +54,95 @@ public String endpointId() {
         return endpointId;
     }
 
-    interface Builder {
-        Builder region(String region);
+    public Boolean defaultTrueParam() {
+        return defaultTrueParam;
+    }
+
+    public String defaultStringParam() {
+        return defaultStringParam;
+    }
+
+    public String deprecatedParam() {
+        return deprecatedParam;
+    }
+
+    public interface Builder {
+        Builder region(Region region);
 
         Builder useDualStackEndpoint(Boolean useDualStackEndpoint);
 
         Builder useFIPSEndpoint(Boolean useFIPSEndpoint);
```

Comment:
useFipsEndpoint?

Reply:
These are taken directly from the rules model--rule authors have the flexibility to name params however they want ant map them to any builtin. For example another service could map just "fips" to the FIPS builtIn.

Reply:
Nm Just read your comment again, you're pointing out the pascal casing. I'll change 👍

Reply:
Yeah, not sure if we can change it easily though if we are using the same code to generate service pojo setters. It's not a blocker

Reply:
Yep it's changed